source/containers.tex

```diff
@@ -12937,20 +13955,25 @@
     explicit stack(const Container&);
     explicit stack(Container&&);
     template<class InputIterator> stack(InputIterator first, InputIterator last);
+    template<@\exposconcept{container-compatible-range}@<T> R> stack(from_range_t, R&& rg);
     template<class Alloc> explicit stack(const Alloc&);
     template<class Alloc> stack(const Container&, const Alloc&);
     template<class Alloc> stack(Container&&, const Alloc&);
     template<class Alloc> stack(const stack&, const Alloc&);
     template<class Alloc> stack(stack&&, const Alloc&);
     template<class InputIterator, class Alloc>
       stack(InputIterator first, InputIterator last, const Alloc&);
+    template<@\exposconcept{container-compatible-range}@<T> R, class Alloc>
+      stack(from_range_t, R&& rg, const Alloc&);
 
```

Comment:
You also removed empty lines from `queue`.
```suggestion
```

Reply:
We should keep both, I think. An empty line between ctors and other methods is beneficial for clarity.

Reply:
We should keep both, I think. An empty line between ctors and other methods is beneficial for clarity.

Reply:
Agreed with @Mick235711.